README.md

```diff
@@ -46,8 +46,9 @@ Afterwards, `nix run .#services-full` will recreate a default development enviro
 We'd love to see contributions! PRs solving existing issues are most helpful to us. It's best if you ask to be assigned for the issue so we won't have multiple people working on the same issue. Feel free to open issues for bugs, setup problems, or feature requests. If you have other questions, feel free to contact the [organization members](https://github.com/orgs/e-valuation/people). You should probably branch off `main`, the branch `release` is used for stable revisions.
 
 Before committing, run the following commands:
-- `./manage.py test` (runs the test suite)
-- `./manage.py lint` (runs linting)
+- `./manage.py typecheck`
+- `./manage.py test`
+- `./manage.py lint`
 - `./manage.py format` (applies automatic code formatting)
 
 or, to combine all three, simply run `./manage.py precommit`.
```

Comment:
```suggestion
or, to combine them, simply run `./manage.py precommit`.
```

Maybe we should swap these and mention the precommit command first and then the list, idk